### ISSUE TITLE
Odd Resolution Error

### ISSUE THREAD

**Issue by RamblingGeekUK** [MichaCo/DnsClient.NET#8]
Locally bbc.co.uk resolves on my PC, doesn't on you page http://dnsclient.michaco.net/  any ideas why?

**Comment by MichaCo** [comment]
Hey @RamblingGeekUK, I had that issue with some domain names from time to time. It is not reproduceable locally. Seems to be an issue with the Azure network/public DNS servers. Not sure exactly why that is...

**Comment by MichaCo** [comment]
@RamblingGeekUK I was able to reproduce it now, I tried "www.bbc.co.uk" which works with the "www".

If I query only `A` records, it works. The `ANY` request though responds with a larger message, > 512. That seems to be the reason why it fails on Azure. Maybe there is a firewall rule or something preventing that. Also, the google servers response for supported message size is always 512, although it totally returns larger ones, too.

When I switch DnsClient to use the TCP protocol only, it just works all the time.

Not sure why that gets blocked, maybe because the message size is larger than expected which could be malicious? idk... But I'm pretty sure this is not an issue with this library ^^
